Userland/Libraries/LibPDF/Parser.cpp

```diff
@@ -374,6 +374,8 @@ String Parser::parse_hex_string()
                     return builder.to_string();
                 }
 
+                if (ch == ' ' || ch == '\t' || ch == '\n')
+                    continue;
```

Comment:
I think this should be replaced by a call to `m_reader.consume_whitespace()` at the start of the loop, because we always read two characters to form a byte and this would increment the counter as well.

Reply:
Thanks, I'm not too familiar with the whole API so I didn't spot that, I've replaced it now as suggested.